torchvision/datapoints/_bounding_box.py

```diff
@@ -10,12 +10,35 @@
 
 
 class BoundingBoxFormat(StrEnum):
+    """[BETA] Coordinate format of a bounding box.
```

Comment:
I thought we wanted to remove the `StrEnum` part? Or do I misremember?

Reply:
Remove what. The class or the functionality to instantiate from a str? IIRC yes for the former and no for the latter. If that is right, should we do it in a follow-up PR?

Reply:
yeah just the inheritance from `StrEnum` - I love using strings lol. And yes, this is for another PR